### DISCUSSION TITLE
没有config.yml怎么配置？

### DISCUSSION THREAD

**Discussion by Satoing** [ppoffice/hexo-theme-icarus#947] [General]
没有config.yml怎么自定义配置？

**Comment by ppoffice** [comment]
https://ppoffice.github.io/hexo-theme-icarus/Configuration/icarus%E7%94%A8%E6%88%B7%E6%8C%87%E5%8D%97-%E4%B8%BB%E9%A2%98%E9%85%8D%E7%BD%AE/#%E9%85%8D%E7%BD%AE%E7%94%9F%E6%88%90%E4%B8%8E%E6%A0%A1%E9%AA%8C
